### ISSUE TITLE
allow users to post an event to their instagram page

### ISSUE THREAD

**Issue by joebeernink** [joebeernink/TrashMob#206]
Add button to event details page to allow users to share event to their instagram page (assuming this is possible)

**Comment by joebeernink** [comment]
Not currently possible with current instagram SDK

**Comment by chunlishao** [comment]
@joebeernink , I agree with you, the current instagram graph api is for business and creator account to publish post. the basic api is for get user's profile and posts information. None of them work for us.

For the mobile apps, Instagram provides the "sharing to feed" and "sharing to story" api for third party apps. The way is we have picture or video prepared, and launches the instagram app and pass it to instagram then post manually.

I think it would be a fantastic feature to have after allowing user to upload photos and videos regarding event cleaning. I can build a prototype later if needed.
